libraries/chain/block_header_state.cpp

```diff
@@ -7,6 +7,11 @@
 
 namespace eosio::chain {
 
+// moved this warning out of header so it only uses once
+#warning TDDO https://github.com/AntelopeIO/leap/issues/2080
```

Comment:
Now it occurs to me, if we already have an issue, we don't need a warning, because the issue will remind us.